### PR TITLE
Hotfix/multiple pipe diagram

### PR DESCRIPTION
**Pull request by michaelgruner**

### CODE REVIEW

File: plugins/tracers/gstdot.c [hunk 1]
```diff
@@ -70,8 +74,10 @@ gst_dot_pipeline_to_file (GstBin * bin, GstDebugGraphDetails flags)
         full_trace_dir);
   }
 
-  full_file_name = g_strdup_printf ("%s" G_DIR_SEPARATOR_S "pipeline.dot",
-      full_trace_dir);
+  strftime (time_suffix, MAX_SUFFIX_LEN, "%F_%T", localtime (&now));
```
Comment (bill9889) [inline plugins/tracers/gstdot.c]: Why was `strftime` used instead of Glib function `g_date_strftime`?

File: plugins/tracers/gstdot.c [hunk 2]
```diff
@@ -70,8 +74,10 @@ gst_dot_pipeline_to_file (GstBin * bin, GstDebugGraphDetails flags)
...
+
+  full_file_name = g_strdup_printf ("%s" G_DIR_SEPARATOR_S "%s_%s.dot",
```
Comment (bill9889) [inline plugins/tracers/gstdot.c]: Is `full_file_name` being freed below?

File: plugins/tracers/gstdot.c (hunk 1 above)
Comment (michaelgruner) [reply plugins/tracers/gstdot.c]: I didn't find an equivalent. According to `g_date_strftime` documentation:

```
Works just like the platform's C library strftime() function, but only accepts date-related formats; time-related formats give undefined results. 
```

File: plugins/tracers/gstdot.c (hunk 2 above)
Comment (michaelgruner) [reply plugins/tracers/gstdot.c]: Yes sir

**Pull request merged**